### PR TITLE
Building a lyrics search app with vanilla javascript

### PR DESCRIPTION
**Pull request by shegz101**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by shegz101** [comment]
@adrianmurage i have fixed the issue

**Comment by adrianmurage** [comment]
👍🏽  @shegz101 I'll be looking over the changes later today

**Review by adrianmurage — CHANGES_REQUESTED** [review]
@shegz101
👍🏽 good progress with the changes

A few things to take note of:
1. You have two author folders:
`/authors/olusegun-bobate/index.md`
`/authors/bobate-olusegun/index.md`
kindly delete one
2. I have highlighted a few things that still need to change while conducting the preliminary review. Kindly make the changes as I proceed to do the technical review.

**Comment by adrianmurage** [comment]
this closes #3990

**Comment by shegz101** [comment]
@adrianmurage do I need to open a new pull request

**Comment by adrianmurage** [comment]
> @adrianmurage do I need to open a new pull request

No, just make changes here

File: content/articles/building-lyrics-search-app-with-vanilla-javascript/index.md [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+           <div id=”lyrics-search”>
+                   <form action=”#” id=”searchMe”>
+                         <input type=”text” id=”lyricSearch” placeholder=”artist name or song title”/>
+                    </form>
+                     <div id=”search-result”></div>
+           </div>
+   </div>
+<script src=”lyrics.js”></script>
+</body>
+</html>
```
Comment (adrianmurage) [inline content/articles/building-lyrics-search-app-with-vanilla-javascript/index.md:49]: this html is buggy
It seems that the quotes in the code changed once you copied the code into your markdown file.

Kindly go over the code and make sure it has the correct quotes

here's what I ended up with when I tried to correct it:
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <link rel="stylesheet" href="lyrics.css" />
    <title>Lyrics Search App</title>
  </head>
  <body>
    <div class="container">
      <div class="intro-text">
        <h1>Learn your favourite</h1>
        <h2>song lyrics</h2>
      </div>
      <div id="lyrics-search">
        <form action="#" id="searchMe">
          <input
            type="text"
            name=""
            id="lyricSearch"
            placeholder="artist name or song title"
          />
        </form>
        <div id="search-result"></div>
      </div>
    </div>
    <script src="lyrics.js"></script>
  </body>
</html>

```

**Comment by shegz101** [comment]
@adrianmurage I have made all neccessary changes

File: content/articles/building-lyrics-search-app-with-vanilla-javascript/index.md [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+const output = document.getElementById(“search-result”);//target the output div
+
+```
+Let’s explore APIs and the desired API we would use to get the data we need to make this web app up and running.
+ 
+Declare the API URL using the code below:
+
+```javascript
+
+const api = “https:/api.lyrics.ovh”;
```
Comment (adrianmurage) [inline content/articles/building-lyrics-search-app-with-vanilla-javascript/index.md:179]:
```suggestion
const api = "https://api.lyrics.ovh";
```

**Review by adrianmurage — CHANGES_REQUESTED** [review]
@shegz101 

Your code seems to have quotation marks that are erratic. I have pointed out a few examples of what I mean by this in the review.

Kindly ensure that all your code can run by:
1. setting up an empty project
2. following through your article as a reader of  your article would. This will allow you to pick up anything that might be dysfunctional in the code you have in your article.

File: content/articles/building-lyrics-search-app-with-vanilla-javascript/index.md (hunk 1 above)
Comment (shegz101) [reply content/articles/building-lyrics-search-app-with-vanilla-javascript/index.md:49]: @adrianmurage , for me to make changes to those quotes,  I would have to open a new PR, but my code editor didn't return errors concerning the quotes, only if you could suggest changes and I commit them

**Comment by shegz101** [comment]
@adrianmurage just checked my code editor, the quotes are wrong, funny how I never noticed this, how can I make changes without this pull request being closed

**Comment by shegz101** [comment]
@adrianmurage i have sorted out the issues of the quotation marks, and I have corrected them with the right quotes

**Comment by adrianmurage** [comment]
> @adrianmurage i have sorted out the issues of the quotation marks, and I have corrected them with the right quotes

@shegz101 I'll take a look at it tomorrow
Have a lovely Sunday evening 👍🏽

**Comment by shegz101** [comment]
@adrianmurage thanks for all you've done..looking forward to your reply tomorrow.

**Review by adrianmurage — CHANGES_REQUESTED** [review]
@shegz101 I have run into errors that prevent me from proceeding with the technical review.
Kindly look over them and make the necessary changes.

Also, I will be closing this PR as the article is not ready for review yet given the multiple occurrences of avoidable errors. 

Kindly setup a blank folder and follow through your article as a reader would to make sure that the article can be followed through from start to finish with all the code running.

I believe that one or two passes where you setup everything from scratch should enable you to catch all the errors I am running into.

I look forward to seeing the ready to review version of this article 👍🏽

**Comment by shegz101** [comment]
@adrianmurage can I know specifically the type of error, if it is the one about undefined, it's from the end of the API I am using

**Pull request closed without merging**